cmd/contour/servecontext.go

```diff
@@ -309,7 +310,8 @@ func (ctx *serveContext) grpcOptions() []grpc.ServerOption {
 		}),
 	}
 	if !ctx.PermitInsecureGRPC {
-		tlsconfig := ctx.tlsconfig()
+		log := logrus.StandardLogger()
```

Comment:
I think that this is another case where we should return the error and have the caller do the logging. So let's make this:
```Go
func (ctx *serveContext) grpcOptions() ([]grpc.ServerOption, error) {
...
}
```

Then the caller can log at an appropriate level.

Reply:
This function  `grpcOptions` do not have any error scenario. So I am skipping to resolve this review comment.